common/src/Model/Brush.cpp

```diff
@@ -894,7 +916,15 @@ namespace TrenchBroom {
         Vec3 Brush::splitFace(const BBox3& worldBounds, const Polygon3& facePosition, const Vec3& delta) {
             return addVertex(worldBounds, facePosition.center() + delta)->position();
         }
-       
+        
+        static std::pair<bool, BrushGeometry> rejectVertexMove() {
```

Comment:
Actually, it may also be a struct with public member variables `success` and `result` or something along those lines. No information hiding is necessary since the entire class / struct is private anyway.